community/io/src/test/java/org/neo4j/graphdb/mockfs/EphemeralFileSystemAbstraction.java

```diff
@@ -1204,6 +1203,12 @@ private ByteBuffer allocate( int sizeIndex )
             }
         }
 
+        private int capacity( int sizeIndex )
+        {
+            return (sizeIndex < SIZES.length) ?
+                           SIZES[sizeIndex] : ((sizeIndex - SIZES.length + 1) * SIZES[SIZES.length - 1]);
```

Comment:
The root problem is even more then different scale step, i do not see why we need to do sizeIndex <-> actual size dance at all; this change is  minimal change for current release. Let talk about that as well :)